firefox_ui_harness/runtests.py

```diff
@@ -102,5 +60,9 @@ def cli(runner_class=FirefoxUITestRunner, parser_class=FirefoxUIOptions):
         sys.exit(10)
 
 
+def cli_update():
+    cli(runner_class=UpdateTestRunner, parser_class=UpdateOptions)
+
+
 if __name__ == '__main__':
     sys.exit(cli())
```

Comment:
See above. We do not only have an entry point for updates, but also for other kind of tests. This is just running the default set of tests, which will be unit, functional, and remote tests.